modules/server/server-launcher/src/main/java/org/ballerinalang/composer/server/launcher/ServerLauncher.java

```diff
@@ -139,6 +139,8 @@ public static void main(String[] args) {
             server.start();
             String uRL = "http://" + config.getHost() + ":" + config.getPort();
             logger.info("Composer started successfully at " + uRL);
```

Comment:
Shall we remove the log infor otherwise do the message duplicate ?

Reply:
It didn't show up in the stdout. Is it logged to a log file? Let me check again.

Reply:
Message is not duplicated in the stdout. logger puts it in a log file I think.
![capture](https://user-images.githubusercontent.com/3872221/35405750-2ae8e26e-022d-11e8-9549-5d7d5b9b3608.PNG)

Reply:
Shall we remove it since it is redundant. it doesn't make sense in log file to have that

Reply:
Sure. Removed in 4e2fe90